aten/src/ATen/native/quantized/cpu/qpool.cpp

```diff
@@ -0,0 +1,201 @@
+#include <ATen/ATen.h>
+#include <ATen/Parallel.h>
+#include <ATen/core/Type.h>
+#include <ATen/core/op_registration/op_registration.h>
+#include <ATen/native/TensorIterator.h>
+#include <ATen/native/cpu/Loops.h>
+#include <ATen/quantized/Quantizer.h>
+
+#include <algorithm>
+#include <vector>
+
+namespace at { namespace native {
+namespace {
+
+/* Creates the output shape using the input parameters for the axis.
+
+Args:
+  input_size, kernel_size, padding, stride: Appropriate parameter
+  ceil_mode: use ceiling instead of floor while computing the output shape.
+*/
+inline int64_t pooling_output_shape(int64_t input_size, int64_t kernel_size,
```

Comment:
can you just include the function from `THNN/generic/pooling_shape.h`?

Reply:
Oh, I didn't know that was an option! :) Will try now!